ui/pages/send/send-content/send-amount-row/amount-max-button/amount-max-button.component.js

```diff
@@ -1,81 +1,47 @@
-import React, { Component } from 'react';
```

Comment:
Nit: Typically for components using the functional style, we drop the `.component` suffix on the filename. It's kinda both a "container" and a "component" (in that sense) in one.

Reply:
https://github.com/MetaMask/metamask-extension/pull/10965/commits/e07a1090253ae13ecaf0551d0c93d3a4d336b111